src/app/containers/ChartbeatAnalytics/index.jsx

```diff
@@ -12,37 +15,52 @@ import {
   getDomain,
   buildSections,
   getType,
+  getTitle,
 } from '../../lib/analyticsUtils/chartbeat';
 
-const ChartbeatAnalytics = () => {
-  const { service } = useContext(ServiceContext);
-  const { env, platform, pageType } = useContext(RequestContext);
+import { getReferrer } from '../../lib/analyticsUtils';
+
+const ChartbeatAnalytics = ({ data }) => {
+  const { service, brandName } = useContext(ServiceContext);
+  const { env, platform, pageType, previousPath, origin } = useContext(
+    RequestContext,
+  );
   const { enabled } = useToggle('chartbeatAnalytics');
 
   if (!enabled) {
     return null;
   }
-
+  const referrer = getReferrer(platform, origin, previousPath);
+  const title = getTitle(pageType, data, brandName);
   const domain = env !== 'live' ? getDomain('test') : getDomain(service);
   const sections = buildSections(service, pageType);
   const cookie = getSylphidCookie();
   const type = getType(pageType);
-  const commonProps = {
+
+  const isAmp = platform === 'amp';
+  const config = {
     domain,
-    type,
     sections,
-    cookie,
-    chartbeatUID,
+    uid: chartbeatUID,
+    title,
+    ...(isAmp && { contentType: type }),
+    ...(!isAmp && { type }),
+    ...(!isAmp && { useCanonical }),
+    ...(!!referrer && { virtualReferrer: referrer }),
```

Comment:
Do we need the double bang syntax here? I think we can just do
```suggestion
    ...(referrer && { virtualReferrer: referrer }),
```